htsget.md

```diff
@@ -37,31 +37,56 @@ Requests to the retrieval API endpoint may be authenticated by means of an OAuth
 
 ## Errors
 
-Non-successful invocations of the API return an HTTP error code, and the response body contains a JSON object (content-type application/json) with the following structure:
+The server MUST respond with an appropriate HTTP status code (4xx or 5xx) when
+an error condition is detected.  In the case of transient server errors, 
+(e.g., 503 and other 5xx status codes), the client SHOULD implement appropriate
+retry logic as specified in [RFC2616&sect;10.5](https://tools.ietf.org/html/rfc2616#section-10.5).
 
+For errors that are specific to the `htsget` protocol, the response body SHOULD be a JSON object
+(content-type `application/json`) providing machine-readable information about the nature 
+of the error, along with a human-readable description. The strucure of this JSON object is 
+described as follows.
+
+### Error Response JSON fields
+
+<table>
+<tr markdown="block"><td>
+`error`  
+_string_
+</td><td>
+The type of error. This SHOULD be chosen from the list below.
+</td></tr>
+<tr markdown="block"><td>
+`message`  
+_string_
+</td><td>
+A message specific to the error providing information on how to debug the problem. Clients 
+MAY display this message to the user.
+</td></tr>
+</table>
+
+The following errors  are defined:
+
+| Error			        | HTTP Status   | Description    |
+| --------------------- |-------------- | ------------------------- |
+| InvalidAuthentication	| 401	        | Authorization provided is invalid |
+| PermissionDenied	    | 403	        | Authorization is required to access the resource |
+| NotFound		        | 404	        | The resource requested was not found |
+| Unable			    | 400       	| The server is unable to fulfill the request |
```

Comment:
I've made the change @jmtcsngr --- I think the objection clock times out after a week or so!